pkg/controller/memcached/memcached_controller_test.go

```diff
@@ -99,6 +100,45 @@ func TestMemcachedController(t *testing.T) {
 		})
 	})
 
+	t.Run("when Memcached CR is scaled and Memcached Deployment exist (changed)", func(t *testing.T) {
```

Comment:
`exists`

Reply:
maybe add sth like already exists to it will be clearer :)

Reply:
changed description